tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1862,8 +1862,8 @@ directory](https://github.com/prettier/prettier/tree/master/editors/vim).
 
 ### Visual Studio Code
 
-Can be installed using the extension sidebar. Search for
-\`Prettier - JavaScript formatter\`.
+Can be installed using the extension sidebar. Search for \`Prettier - JavaScript
+formatter\`.
```

Comment:
Is this linebreak a problem?

Reply:
From [commonmark spec](http://spec.commonmark.org/0.25/#example-314), it shouldn't be a problem.